website/content/en/blog/2024-05-30-cncf-cloud-native-sustainability-week-2024-is-underway-and-why-you-should-care.md

```diff
@@ -0,0 +1,77 @@
+
+### Sneak Peek into CNCF Cloud Native Sustainability Week 2024
+
+This week provides a unique opportunity to actively engage in promoting sustainability in the cloud native and open source space. There are multiple ways to participate, allowing everyone to find the format most suitable to their needs and interests. Whether you prefer in-person meetings for a more direct and interactive experience or virtual sessions for greater flexibility, there's an option for you. Additionally, you can also take the initiative to organize a local meetup where you live, contributing to spreading sustainable practices in your local community.
+
+* In-Person Meetups: Join events and meetups organized by local communities around the world. These meetings provide an opportunity to discuss regional challenges and learn about innovative solutions for sustainability. 
+* Virtual Sessions: Participate in webinars, workshops, and online discussions organized by the TAG ENV to explore topics such as renewable energy and IT resource optimization.  
+* Speak at an Event: Propose a session on sustainability topics at a local or online meetup. This is an opportunity to share your ideas and actively contribute to the sustainability discussions. 
+
+### A Look Back at CNCF Cloud Native Sustainability Week 2023
+
+In 2023, the CNCF Cloud Native Sustainability Week saw significant and diverse participation.
+
+Over 22 meetups were organized in more than 17 countries, involving numerous international speakers and thousands of participants. The events covered a wide range of topics, from renewable energy to IT resource optimization.
+
+Reflecting back on last year’s event, which was also organized for the first time, we see that it was a huge success! Based on the feedback from the community, it contributed to:
+
+* Increased Awareness: Awareness campaigns reached a broader audience, educating on sustainable practices through articles, social media posts, and webinars.  
+* Community Engagement: Active participation facilitated the sharing of knowledge and best practices, creating a network of sustainability supporters.  
+* Successful Initiatives: Specific projects were launched, such as improving IT infrastructures to reduce energy consumption.
+
+If you would like to relive the highlights of the Cloud Native Sustainability Week 2023 and hear some significant testimonials about the event, check out the [following video](https://youtu.be/ezGSYtvQH2c?si=3q-USUPyLVjBq9wH). This video gathers the experiences of participants and organizers, offering an inspiring look at the conducted activities and highlighting the most memorable moments from the event. We’ve also published a blog post wrapping up the CNCF Cloud Native Sustainability Week 2023: [Cloud Native Sustainability Week 2023 - putting sustainability on the community radar](https://tag-env-sustainability.cncf.io/blog/2024-wrap-up-cloud-native-sustainability-week-2023)
+
+## The CNCF Cloud Native Sustainability Week is YOU!
```

Comment:
```suggestion
## The CNCF Cloud Native Sustainability Week needs YOU!
```
The original phrasing doesn't quite sound right. I assume from the next paragraph that the sentiment is asking for help?